tests/common.test.ts

```diff
@@ -13,3 +13,40 @@ test('fsExists function', async () => {
   expect(await common.fsExists('.')).toBe(true);
   expect(await common.fsExists('please-don-t-make-me-fail')).toBe(false);
 });
+
+test('extractBumpDigest function', async () => {
```

Comment:
Thanks for adding the spec here, it makes the new feature easier to understand... Even if it's not the easiest review 😅 

If it's not too much to ask, would it be possible to split this file in 2 commits.

1st commit with introduction of 'extractBumpDigest function', with current behavior.
2nd commit with the change introduced by `doc=hello` digest ?

Reply:
ok

Reply:
I've split the commit in two, but in the end I don't think it's such a good idea because each commit are not independent (they both need each other to work correctly)